addons_tests/test_module/models/hop_dong.py

```diff
@@ -34,11 +34,11 @@ class hop_dong(models.Model):
 
     employee_name = fields.Char(string='Employee Name', required=True, compute='_compute_employee_name')
 
-    employee_role = fields.Char(string='Employee Role')
+    employee_role = fields.Char(string='Employee Role', compute='_get_contract_information')
```

Comment:
Viết thế này trường employee_role có khác gì với employee_role cũ k?

Reply:
trường employee_role sẽ được tính toán dựa trên phương thức _get_contract_information ạ